tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -457,6 +462,11 @@ public string GetTargetFramework()
             return $"net{_major}{_minor}{_patch ?? string.Empty}";
         }
 
+        public void DisableAutomaticInstrumentation()
+        {
+            AutomaticInstrumentationEnabled = false;
+        }
```

Comment:
tiny nit: I'm generally not a fan of "negative" named methods, how do you feel about something like this (plus a change at the callsite obviously). Not passionate about it at all, it's test code!

```suggestion
        public void SetAutomaticInstrumentation(bool enabled)
        {
            AutomaticInstrumentationEnabled = enabled;
        }
```

Reply:
Sounds good to me, fixed in e1fb0f388ba11bd278bc0e4af6f1c1b6abfae7ae